backend/spec/requests/memos_spec.rb

```diff
@@ -28,4 +28,48 @@
       end
     end
   end
+
+  describe 'PUT /memos/:id' do
+    context 'コンテンツが有効な場合' do
+      let(:existing_memo) { create(:memo) }
+      let(:update_memo_params) { { content: '新しいコンテンツ' } }
```

Comment:
update_memoというprefixはなくても伝わるかなと思いました〜
```suggestion
      let(:params) { { content: '新しいコンテンツ' } }
```

Reply:
承知しました！
putのテストという時点で、更新系とわかりますもんね！